packages/salesforcedx-vscode-core/src/util/orgUtil.ts

```diff
@@ -70,28 +89,27 @@ export async function checkForExpiredOrgs(orgList: OrgList) {
       return;
     }
 
-    const defaultOptions = Aliases.getDefaultOptions();
-    const aliases = await Aliases.create(defaultOptions);
-
-    const formattedOrgsToDisplay = orgsAboutToExpire.map((orgAboutToExpire: any) => {
-      const alias = aliases.getKeysByValue(orgAboutToExpire.username);
-      const aliasName = alias.length > 0
-        ? alias.toString()
+    const formattedOrgsToDisplay = (await Promise.all(orgsAboutToExpire.map(async orgAboutToExpire => {
+      const aliasName = orgAboutToExpire.aliases && orgAboutToExpire.aliases.length > 0
+        ? orgAboutToExpire.aliases[0]
         : orgAboutToExpire.username;
 
+      const authFields = await getAuthFieldsFor(orgAboutToExpire.username);
```

Comment:
seems like a bummer we make this call twice...once in the fiter then again in the formatter.  can we cache this data or add it to the org for processing?

Reply:
I thought about this when developing this change.  The issue is that (IMHO) caching this would convolute the code.  Let's discuss at standup today.

Reply:
I changed this to a for loop